pkg/services/rendering/rendering.go

```diff
@@ -296,7 +297,7 @@ func (rs *RenderingService) render(ctx context.Context, renderType RenderType, o
 		return rs.renderUnavailableImage(), nil
 	}
 
-	if renderType == RenderPDF || opts.Encoding == "pdf" {
```

Comment:
I couldn't find a case where we could receive a "pdf" encoding with a different `renderType` of PDF